packages/application-generic/src/usecases/digest-filter-steps/digest-filter-steps.usecase.ts

```diff
@@ -69,6 +69,9 @@ export class DigestFilterSteps {
       const keys = path.split('.');
 
       for (const key of keys) {
+        if (result === undefined) {
```

Comment:
Fix so if you have a nested digestkey but do not pass a nested value it will throw "can not find x on undefined"

Reply:
Great find!